tailwind.config.js

```diff
@@ -3,6 +3,15 @@ export default {
     prefix: "",
     theme: {
         extend: {
+            colors: {
+                primary: {
+                    DEFAULT: "#3C8786",
+                    dark: "#2B6E74",
+                    darkest: "#1E5661",
+                    light: "#6AB7AF",
+                    lightest: "#E8F5F5",
```

Comment:
Only `DEFAULT` and `darkest` atm. Probably worth it as a near short term project to create and covert to a tailwind style pallet for primary before this gets used too widely. 

I used the old color choices now to prevent incongruity between the brightness levels where the `primary` versions are used

Reply:
Is the darker variant only used in the side nav bar? We should just change those to blue IMO.